vendor/github.com/nats-io/nuid/LICENSE

```diff
@@ -1,201 +1,21 @@
-                                 Apache License
```

Comment:
This is strange.. looks like it removed Apache license and went back to MIT license?

Reply:
hmm let me look I see it too..

Reply:
Might not have tagged latest, modules seem to like releases..